### DISCUSSION TITLE
关于笔记下面添加定位信息

### DISCUSSION THREAD

**Discussion by yzzgchina** [Tokinx/Wing#122] [想法]
能不能在发送笔记的时候定位下当前地址，并显示在下面呢，这样就跟微博基本上无差距了~

**Comment by Tokinx** [comment]
可能存在泄露隐私的风险，同时获取位置信息并展示具体的标志需要（如：百度地图、高德地图）API，这需要每一位使用者都申请一个API的许可，可能会增加主题使用难度，暂时可能不会考虑这种方式。
可能会考虑展示不同渠道推送的笔记（如：Send by Shortcut、Send by Browser plugin），并支持自定义
当然这些只是初步的设想，开发进度还是比较缓慢的。。。
